packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import debounce from 'lodash/debounce';
+import { css } from 'emotion';
 import { DataFrame, CSVConfig, readCSV } from '@grafana/data';
 import { Icon } from '../Icon/Icon';
+import { GrafanaTheme } from '@grafana/data';
+import { Themeable } from '../../types/theme';
+import { TextArea } from '../TextArea/TextArea';
+import { stylesFactory } from '../../themes';
+import { withTheme } from '../../themes/index';
 
-interface Props {
+interface Props extends Themeable {
   config?: CSVConfig;
   text: string;
   width: string | number;
   height: string | number;
   onSeriesParsed: (data: DataFrame[], text: string) => void;
 }
 
+interface TableInputCSVStyle {
+  tableInputCsv: string;
+  textarea: string;
+  footer: string;
+}
+
```

Comment:
nit: move this closer to the getStyles function. This will make it easier to read the props for the Component.